fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -16,6 +16,10 @@ def running_init_command?
         ARGV.include?("init")
       end
 
+      def utf8_locale?
+        (ENV['LANG'] || "").end_with?("UTF-8", "utf8") || (ENV['LC_ALL'] || "").end_with?("UTF-8", "utf8") || (FastlaneCore::CommandExecutor.which('locale') && `locale charmap` == "UTF-8")
```

Comment:
```suggestion
        (ENV['LANG'] || "").end_with?("UTF-8", "utf8") || (ENV['LC_ALL'] || "").end_with?("UTF-8", "utf8") || (FastlaneCore::CommandExecutor.which('locale') && `locale charmap`.strip == "UTF-8")
```

`locale charmap` on macos (at least on my machine) returns `"UTF-8\n"` so I think we need to a `.strip` to this just for safety 🤔

Reply:
Thanks a lot for the suggestion.